components/CurrentSong.vue

```diff
@@ -34,6 +46,9 @@ export default Vue.extend({
 
       return artists.map(({ name }) => name).join(', ');
     },
+    getBackgroundColor() {
+      return sessionStorage.getItem('color');
```

Comment:
If we store the color in `sessionStorage` we cannot apply the change in OBS, we should move it to the query string